src/__tests__/api-negative.test.ts

```diff
@@ -0,0 +1,299 @@
+import { PaDB } from 'src/types';
+import request from 'supertest';
+
+const APILink = 'http://localhost:4000';
+// const validateuuid = (id: string) => {
+//   const uuidRegExp =
+//     /^[0-9a-fA-F]{8}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{4}\b-[0-9a-fA-F]{12}$/gi;
+//   return uuidRegExp.test(id);
+// };
+
+// const testDB = []
+
+describe('Test negative cases', () => {
+  const endpoint = '/api/users';
+  describe('wrong requests', () => {
+    // const link = endpoint + '/' + 'b92a1ae3-4e54-4246-87b9-002472915bd6';
+    describe('wrong GET request', () => {
+      test('empty GET request', async () => {
+        const response = await request(APILink).get('');
+        expect(response.statusCode).toBe(404);
+        expect(response.body).toBe('Route not found');
+      });
+
+      test('wrong GET request', async () => {
+        const response = await request(APILink).get('/unknown');
+        expect(response.statusCode).toBe(404);
+        expect(response.body).toBe('Route not found');
+      });
+
+      test('invalid id, GET', async () => {
+        const response = await request(APILink).get('/api/users/1');
+        expect(response.statusCode).toBe(400);
+        expect(response.body).toBe('User ID is invalid');
+      });
+    });
+
+    describe('wrong POST request', () => {
+      test('empty POST request', async () => {
+        const response = await request(APILink).post(endpoint).send({});
+        expect(response.statusCode).toBe(400);
+        expect(response.body).toBe('Request body does not contain reqied fields');
```

Comment:
Can be replaced with a const:
```suggestion
    expect(response.body).toBe(CommonErrors.noRecsReq.message);
```